Crashlytics/ProtoSupport/Protos/crashlytics.proto

```diff
@@ -33,9 +34,16 @@ message Report {
   // General device type which generated the Event
   Platforms platform = 4;
 
-  // Unique device generated guid for application install.
+  // Unique Crashlytics-specific device generated guid for application install.
+  // Equivalent to Session.app.installation_uuid
   string installation_uuid = 5;
 
+  // Unique device generated ID from the FirebaseInstallations SDK
+  string firebase_installation_id = 16;
+
+  // The last Session ID associated with the crash report.
+  string app_quality_session_id = 17;
```

Comment:
Would the backend change be done in a separate CL?

Reply:
Yes that is done in cl/499957188